service/src/main/java/com/myhome/configuration/EmailTemplateConfig.java

```diff
@@ -0,0 +1,63 @@
+package com.myhome.configuration;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.support.ResourceBundleMessageSource;
+import org.thymeleaf.spring5.SpringTemplateEngine;
+import org.thymeleaf.templateresolver.ClassLoaderTemplateResolver;
+import org.thymeleaf.templateresolver.ITemplateResolver;
+
+import java.util.Locale;
+
+@Configuration
+public class EmailTemplateConfig {
+
+  @Value("${spring.mail.templates.path}")
```

Comment:
Let's wrap it with configuration properties and have like `MailTemplateProperties` nad `LocalizationProperties`.
See: 
https://www.baeldung.com/configuration-properties-in-spring-boot